plsql/PlSqlParser.g4

```diff
@@ -833,7 +836,7 @@ foreign_key_clause
     ;
 
 references_clause
-    : REFERENCES tableview_name paren_column_list
+    : REFERENCES table_referenced=tableview_name paren_column_list
```

Comment:
What the sense of using element label here? `tableview_name` accessible obviously from visitor.